app/api/api_v1/api.py

```diff
@@ -0,0 +1,12 @@
+"""
+Import all routers together
+"""
+from fastapi import APIRouter
+from app.api.api_v1.routes import healthcheck
+
+api_router = APIRouter()
+
+
+api_router.include_router(
+    healthcheck.router, prefix="/healthcheck", tags=["healthcheck"]
```

Comment:
Move this to top-level if we're good with removing the `/api/` prefix?

Reply:
See comment below. If it's all right I would like to leave the structure as is based on most of the examples I have read. If that is not okay, it can definitely be changed.